src/Linux/Avalonia.LinuxFramebuffer/LinuxFramebufferPlatform.cs

```diff
@@ -112,9 +112,9 @@ public void Shutdown(int exitCode)
 
 public static class LinuxFramebufferPlatformExtensions
 {
-    public static int StartLinuxFbDev<T>(this T builder, string[] args, string fbdev = null, double scaling = 1)
+    public static int StartLinuxFbDev<T>(this T builder, string[] args, string fbdev = null, PixelFormat? format = null, double scaling = 1)
```

Comment:
Same problem here, breaks binary compatibility.

Reply:
Shall I create a second StartLinuxFbDev overload and mark the original as obsolete?

```
    [Obsolete("Use the alternative StartLinuxFbDev extension method")]
    public static int StartLinuxFbDev<T>(this T builder, string[] args, string fbdev = null, double scaling = 1)
        where T : AppBuilderBase<T>, new() =>
        StartLinuxDirect(builder, args, new FbdevOutput(fileName: fbdev, format: null) { Scaling = scaling });
    public static int StartLinuxFbDev<T>(this T builder, string[] args, string fbdev = null, PixelFormat? format = null, double scaling = 1)
        where T : AppBuilderBase<T>, new() =>
        StartLinuxDirect(builder, args, new FbdevOutput(fileName: fbdev, format: format) { Scaling = scaling });
```

Reply:
Forget that, it will cause an ambiguous call at compile time for existing apps.
Perhaps, name the new extension method ```StartLinuxFbDevice```?
Any preferences?

Reply:
```cs

    public static int StartLinuxFbDev<T>(this T builder, string[] args, string fbdev = null, double scaling = 1);
    public static int StartLinuxFbDev<T>(this T builder, string[] args, string fbdev, PixelFormat? format, double scaling);
```

This one shouldn't cause issues. We can remove an extra overload just before the next release.

Reply:
Done